cfgov/unprocessed/apps/youth-employment-success/js/route.js

```diff
@@ -13,6 +13,10 @@ function createRoute( route = {} ) {
     daysPerWeek: '',
     miles: '',
     dailyCost: '',
+    // these two cost values are null to avoid having
+    // a selection made on the front end.
```

Comment:
Maybe clarify this comment slightly. Not sure I understand what would happen if this had a value. Like if these had a value a cost calculation is made before user input?